packages/experiments/package.json

```diff
@@ -20,6 +20,7 @@
     "update-snapshots": "node ../../scripts/build.js jest -u"
   },
   "devDependencies": {
+    "@types/deep-assign": "^0.1.1",
```

Comment:
Technically this should be listed as a dependency. Or, do we already have a merge in utilities?

Reply:
Yeah, I feel like since this is used by running code it shouldn't be just a dev dependency